osmnames/import_osm/import_osm.py

```diff
@@ -0,0 +1,92 @@
+import glob
+import os
+from subprocess import check_call
+from osmnames.helpers.database import psql_exec, exec_sql_from_file
+from osmnames import settings
+
+
+def run():
+    download_pbf()
+    import_pbf_files()
+    create_helper_tables()
+    create_functions()
+    prepare_imported_data()
+
+
+def download_pbf():
+    url = settings.get("PBF_URL")
+    destination_dir = settings.get("IMPORT_DIR")
+    check_call(["wget", "--no-clobber", "--directory-prefix", destination_dir, url])
+
+
+def import_pbf_files():
+    import_dir = settings.get("IMPORT_DIR")
+    pbf_files = glob.glob("{}/*.pbf".format(import_dir))
+
+    if len(pbf_files) == 0:
+        raise IOError("No PBF files for import found in path {}.".format(import_dir))
+
+    for pbf_file in pbf_files:
+        import_pbf_file(pbf_file)
+
+
+def import_pbf_file(pbf_file):
+    imposm_connection = "postgis://{}@{}/{}".format(settings.get("DB_USER"),
+                                                    settings.get("DB_HOST"),
+                                                    settings.get("DB_NAME"))
+
+    check_call(["imposm3", "import",
```

Comment:
I think this would be a nicer format:

```python
check_call([
    "imposm3", "import",
    "-connection", imposm_connection,
    "-mapping", "{}/mapping.yml".format(settings.get("IMPORT_DIR")),
    "-dbschema-import", settings.get("DB_SCHEMA"),
    "-read", pbf_file,
    "-write",
    "-overwritecache",
])```